python/GafferUI/ColorChooser.py

```diff
@@ -620,6 +648,29 @@ def colorChangedSignal( self ) :
 
 		return self.__colorChangedSignal
 
+	## A signal emitted whenver the visible components are changed. Slots
```

Comment:
whenver -> whenever

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/6003/commits/9ec28a6888470afd72b38d584d3b44a8b564b69b